packages/roosterjs-editor-plugins/lib/plugins/Paste/sourceValidations/isExcelDesktopDocument.ts

```diff
@@ -1,9 +1,8 @@
-import { PROG_ID_NAME } from './constants';
 import type { getSourceFunction, getSourceInputParams } from './getPasteSource';
 
-const EXCEL_ATTRIBUTE_NAME = 'xmlns:x';
+// Excel Online does not have this attribute
+export const EXCEL_DESKTOP_ATTRIBUTE_NAME = 'xmlns:x';
```

Comment:
Since we already have a constant.ts, you can put this constant there.

Also please add comment "@internal" to avoid generating document for this export